model/event/task_event_test.go

```diff
@@ -102,3 +105,13 @@ func TestLoggingTaskEvents(t *testing.T) {
 		})
 	})
 }
+
+func TestLogManyTestEvents(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+	require.NoError(db.ClearCollections(AllLogCollection))
+	LogManyTaskAbortRequests([]string{"task_1", "task_2"}, evergreen.GithubPRRequester)
+	events := []EventLogEntry{}
+	assert.NoError(db.FindAllQ(AllLogCollection, db.Query(bson.M{}), &events))
+	assert.Len(events, 2)
```

Comment:
Can you test the data in at least one of the events?

Reply:
done